### PR TITLE
Spelling

### PR DESCRIPTION
**Pull request by jsoref**
Generated by https://github.com/jsoref/spelling `f`; to maintain your repo, please consider `fchurn`

### CODE REVIEW

File: tests/test_octodns_provider_dyn.py [hunk 1]
```diff
@@ -932,14 +932,14 @@ def test_mod_geo_create(self, mock):
         provider = DynProvider('test', 'cust', 'user', 'pass',
                                traffic_directors_enabled=True)
 
-        # will be tested seperately
+        # will be tested seperatelyly
```
Comment (yzguy) [inline tests/test_octodns_provider_dyn.py]: `separately` would be the correct spelling for this one.

File: tests/test_octodns_provider_cloudflare.py [hunk 2]
```diff
@@ -599,7 +599,7 @@ def test_cdn(self):
         zone = Zone('unit.tests.', [])
         provider.populate(zone)
 
-        # the two A records get merged into one CNAME record poining to the CDN
+        # the two A records get merged into one CNAME record pointing to the CDN
```
Comment (yzguy) [inline tests/test_octodns_provider_cloudflare.py]: `tests/test_octodns_provider_cloudflare.py:602:80: E501 line too long (80 > 79 characters)`
Error causing the test failure

File: tests/test_octodns_provider_googlecloud.py [hunk 3]
```diff
@@ -401,8 +401,8 @@ def test__get_gcloud_zone(self):
         provider.gcloud_client.list_zones = Mock(
             return_value=DummyIterator([]))
 
-        self.assertIsNone(provider.gcloud_zones.get("nonexistant.xone"),
-                          msg="Check that nonexistant zones return None when"
+        self.assertIsNone(provider.gcloud_zones.get("nonexistent.xone"),
```
Comment (yzguy) [inline tests/test_octodns_provider_googlecloud.py]: We might want to take this change to make this `nonexistent.zone`, instead of `.xone`

**Comment by yzguy** [comment]
After fixing the issue caught by the linter, the tests all pass. Just a couple comments inline

File: tests/test_octodns_provider_googlecloud.py (hunk 3 above)
Comment (jsoref) [reply tests/test_octodns_provider_googlecloud.py]: Will do

**Comment by jsoref** [comment]
I'll try to get to this sometime this week...

**Comment by yzguy** [comment]
Thanks! Great little tool you have there for fixing up simple spelling mistakes!

File: tests/test_octodns_provider_dyn.py (hunk 1 above)
Comment (jsoref) [reply tests/test_octodns_provider_dyn.py]: This was me trying to do a fix by hand instead of letting my browser suggest the spelling fix. Doh.

**Comment by yzguy** [comment]
@ross looks good to me. Do you think it’s worth squashing down to 1 commit for all the changes?

**Comment by jsoref** [comment]
FWIW, I leave them as individual commits while developing because it makes addressing errors, dropping things which people want to keep, and dealing w/ merge conflicts during rebase much easier.

**Comment by ross** [comment]
> @ross looks good to me. Do you think it’s worth squashing down to 1 commit for all the changes?

Yeah. The history will remain here if anyone is interested in it and that'll avoid a load of small commits in `git log` etc.

If you're both happy with the state of it. I'll merge.

**Comment by jsoref** [comment]
Note that I tend to delete my branches, so the individual commits won't last. If you need me not to, I can...

**Comment by ross** [comment]
> Note that I tend to delete my branches, so the individual commits won't last. If you need me not to, I can...

Pretty sure they'll stick around anyway (thanks to behind the scenes GH magic) so it should be fine to delete.

**Comment by yzguy** [comment]
Loooks good to me!

**Pull request merged**